resources/js/Pages/Notes/Index.vue

```diff
@@ -0,0 +1,61 @@
+<script setup>
+import { defineProps, ref } from 'vue'
+import { Inertia } from '@inertiajs/inertia'
+import { TrashIcon  } from '@heroicons/vue/24/solid'
+const props = defineProps({
+  notes: Array,
+})
+
+const noteText = ref('')
+
+const saveNotes = () => {
```

Comment:
There is no need to make like that. We can delete this method, and for create button set the link to POST request to create a note.

Reply:
Or we can leave it as it is, just we have to stick to one option in whole project.